docs/how-to-guides/deploy-a-flow/distribute-flow-as-executable-app.md

```diff
@@ -0,0 +1,251 @@
+# Distribute flow as executable app
+:::{admonition} Experimental feature
+This is an experimental feature, and may change at any time. Learn [more](../faq.md#stable-vs-experimental).
+:::
+
+We are going to use the [web-classification](https://github.com/microsoft/promptflow/tree/main/examples/flows/standard/web-classification/) as
+an example to show how to distribute flow as executable app with [Pyinstaller](https://pyinstaller.org/en/stable/requirements.html#).
+
+
+Please ensure that you have installed all the required dependencies. You can refer to the "Prerequisites" section in the README of the [web-classification](https://github.com/microsoft/promptflow/tree/main/examples/flows/standard/web-classification/) for a comprehensive list of prerequisites and installation instructions. And we recommend you to add a `requirements.txt` to indicate all the required dependencies for each flow. 
+
+We use [Pyinstaller](https://pyinstaller.org/en/stable/installation.html) to package the flow and [streamlit](https://docs.streamlit.io/library/get-started) to create custom web apps. So, Please install them first before distributing the flow.
+
+
+## Build a flow as executable format
+Note that all dependent connections must be created before building as executable.
+```bash
+# create connection if not created before
+pf connection create --file ../../../examples/connections/azure_openai.yml --set api_key=<your_api_key> api_base=<your_api_base> --name open_ai_connection
+```
+
+Use the command below to build a flow as executable format:
+```bash
+pf flow build --source <path-to-your-flow-folder> --output <your-output-dir> --format executable
+```
+
+## Executable format folder structure
+
+Exported files & its dependencies are located in the same folder. The structure is as below:
+- flow: the folder contains all the flow files.
```

Comment:
Highlight the element name `flow`

Reply:
you mean `flow`: the folder contains all the flow files. ?

I just keep the format as other deploy doc.